aimsAndScope.md

```diff
@@ -1,10 +1,22 @@
 # Aims and scope
 
-*Journal of Cheminformatics* is an open access journal publishing original peer-reviewed research in all
+*Journal of Cheminformatics* is an open science journal publishing original peer-reviewed research in all
```

Comment:
Maybe consider writing "open access, open science"? One describes the jounal publishing model, the other what the journal strives to achieve. Open science does include both implicitly ...

Reply:
Agree - useful to specifically address Open Access and Open Science

(even if one implies the other, it's not obvious to the casual reader, so it's good to be explicit)